modules/base/src/main/scala/io/janstenpickle/trace4cats/base/context/Lift.scala

```diff
@@ -20,4 +20,11 @@ trait Lift[Low[_], F[_]] extends ContextRoot { self =>
 
 object Lift {
   def apply[Low[_], F[_]](implicit ev: Lift[Low, F]): Lift[Low, F] = ev
+
+  trait IdLift[F[_]] extends Lift[F, F] {
+    def Low = F
+    def lift[A](fa: F[A]): F[A] = fa
+  }
+
+  implicit def idLift[F[_]](implicit M: Monad[F]): Lift[F, F] = new IdLift[F] { val F = M }
```

Comment:
Introduction of new derivators should be checked via summon tests. These two new derivators overlaps as `Lift[F, F]`. This propabably should be moved to `ContextRoot`.

Reply:
I can add summon tests, but I'll wait until we settle on a name (if not `Id`) first to avoid needing to rename the tests as well.

Re: overlapping instances, the compiler properly resolves implicit instances so I'm not sure I follow the need to move one or the other.

![](https://user-images.githubusercontent.com/4718399/163425357-22c6548d-9747-48ff-96db-91c926440a9a.png)